bundles/SeoBundle/src/Redirect/RedirectHandler.php

```diff
@@ -38,13 +37,15 @@
 /**
  * @internal
  */
-final class RedirectHandler implements LoggerAwareInterface
+final class RedirectHandler
 {
-    use LoggerAwareTrait;
     use RecursionBlockingEventDispatchHelperTrait;
 
     const RESPONSE_HEADER_NAME_ID = 'X-Pimcore-Redirect-ID';
 
+    private ?LoggerInterface $logger = null;
+    private ?LoggerInterface $redirectLogger = null;
```

Comment:
Can't be nullable, because it is always set in the constructor

Reply:
@blankse Thanks for reporting it :) It will be fixed by https://github.com/pimcore/pimcore/pull/15177